### ISSUE TITLE
Hyperparameter issues in configs

### ISSUE THREAD

**Issue by molereddy** [locuslab/tofu#11]
I'm not sure the hyperparameters in the config files match the reported ones in the paper.
1. for example https://github.com/locuslab/tofu/blob/main/config/forget.yaml#L9 uses 1% forget set, but https://github.com/locuslab/tofu/blob/main/config/forget.yaml#L20C23-L20C44 in the same file uses a 90% retain set. 
3. https://github.com/locuslab/tofu/blob/main/config/forget.yaml#L13 uses 10 epochs while the paper says all experiments are done with 5 epochs.

I also question if it makes sense that for every kind of loss, we use the same number of epochs. These losses have very different natures and different stabilities. This would, imo, lead to unnecessary overfitting for some methods.

**Comment by molereddy** [comment]
The readme has different hyperparameters as well (lr=2e-5 instead of 1e-5 as in the paper and elsewhere in the code).
Also in the data/ directory we have json splits only for wd 0.01 but 0.0 is also tried. I would also appreciate a clarification about what these splits are for and how they are used.

**Comment by molereddy** [comment]
Thank you for the clarifications!
